vignettes/teal-options.Rmd

```diff
@@ -66,14 +66,14 @@ This option controls the dots per inch of the graphs rendered and downloaded whe
 Default: 72
 
 ### `teal.log_layout` (`character`)
-This defines the layout of a log message used in a `teal` application. `teal` uses this layout to format the emitted log messages. Read the documentation of `teal.logger::register_logger` for more information.
+This defines the layout of a log message used in a `teal` application. `teal` uses this layout to format the emitted log messages. Read the documentation of `teal.logger::register_logger` for more information. This must be set before loading any `teal` package(s).
```

Comment:
@pawelru I would rather say that:

If you set those options, you need to load `teal` package again if it was already loaded.

Reply:
Hmmm... How would you phrase it?

I'm thinking about the use cases, i.e. how this will used. To me, most of the people will be creating some kind of project structure or even a simple `app.R` file and I want them to instruct how to order stuff. This is slightly different to how we are working, that is: interactive session with a lots of load/attach on-the-fly. My point is that in that project structure it's quite uncommon (or I would even say: not recommended) to reload packages. WDYT?

Reply:
> If you set those options, you need to load `teal` package again if it was already loaded.

That can't be done without detaching, can it? `library` exits early if the package is already attache, I suppose it is the same for loading.

Reply:
`Recommended to set before loading teal package(s). If packages were loaded first, and the option was set after, please reload the package to enable the option.`

?

Reply:
> > If you set those options, you need to load `teal` package again if it was already loaded.
> 
> That can't be done without detaching, can it? `library` exits early if the package is already attache, I suppose it is the same for loading.

Yes of course. This would have to be detaching and attached again.

Coming back to my previous thought - I actually don't really mind. The documentation is also for developers and they are oftentimes working in a "more interactive" way. I think I will apply your suggestion